devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy

```diff
@@ -97,9 +86,24 @@ def configure() {
 			//Makes Fibaro KeyFob buttons send all kind of supported events
 		}
 	}
+	setupChildDevices()
 	cmds
 }
 
+def setupChildDevices(){
+	def numberOfButtons = prodNumberOfButtons[zwaveInfo.prod]
+	sendEvent(name: "numberOfButtons", value: numberOfButtons, displayed: false)
+	if(!childDevices) {
+		addChildButtons(numberOfButtons)
+	}
+	if(childDevices) {
+		for(def endpoint : 1..prodNumberOfButtons[zwaveInfo.prod]) {
+			def event = createEvent(name: "button", value: "pushed", isStateChange: true)
+			sendEventToChild(endpoint, event)
+		}
```

Comment:
this code should only run if the device didn't initially have the child devices. This will send a "pushed" event to every child every time configure is called

Reply:
Ok, done